### DISCUSSION TITLE
how to edit the linked url of the version buttons in the docVersionDropdown ?

### DISCUSSION THREAD

**Discussion by AkashKumar7902** [facebook/docusaurus#9315] [Q&A] [answered]
![image](https://github.com/facebook/docusaurus/assets/91385321/7b8f17d6-ee30-4cd6-a13a-6e315d30c598)
I want to change the linked url of 2.0.0 version button. currently according to the [docusaurus docs](https://docusaurus.io/docs/api/themes/configuration#navbar-docs-version-dropdown) :

>The user will be able to switch from one version to another, while staying on the same doc

I want to navigate to a different doc instead of the same doc when version is switched. Is this possible ?

**Comment by slorber** [comment, marked as answer]
> I want to navigate to a different doc instead of the same doc when version is switched. Is this possible ?

We don't provide any other behavior.
If you want to implement one you can swizzle the versioning dropdown component and customize it according to your needs

**Reply by AkashKumar7902** [reply]
@slorber How can we do that ?

**Reply by slorber** [reply]
You can read the doc and learn: https://docusaurus.io/docs/swizzling

Otherwise you can pay someone to do it for you: https://github.com/facebook/docusaurus/discussions/9281
